scripts/fix_genesis.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
-if [ "$1" == "--help" ] ; then
+set -e
+if ([ "$1" == "--help" ] || [ -z "$2" ]); then
```

Comment:
I knew something was sus with `()`, as I don't remember ever using that in `if` statements in my previous life of writing bash scripts. Refreshing my memory, that creates subshell, which isn't necessary but is a performance hit.
```suggestion
if [ "$1" == "--help" ] || [ -z "$2" ]; then
```